single-user/Dockerfile

```diff
@@ -46,6 +47,24 @@ RUN mamba install -y --quiet octave_kernel=0.32.0 \
     && jupyter kernelspec install "$SITE_PACKAGES/octave_kernel" --name octave --prefix /opt/conda \
     && conda clean --all
 
+# Rstudio
+USER root
```

Comment:
there is already here https://github.com/EGI-Federation/egi-notebooks-images/blob/4f60eea8750d8e3576d27f2d6cdbb111803006cd/single-user/Dockerfile#L93 the installation of some packages with apt-get, maybe the 2 could be combined

Reply:
was just discussing earlier with @mrorro about this becoming a bit unmanageable.

For clarity of where each things comes from I would keep them separated - this was installed for an EMSO tutorial, and may not make sense anymore to have.

Reply:
also I'm realising this apt-get install does not make sense :( as this package is also available in conda. it would have been cleaner to have this just installed as part of the previous line

Reply:
I'm pushing a change to remove the second apt-get but this should be moved out of the image to CVMFS